packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2269,14 +2273,16 @@ class RenderFittedBox extends RenderProxyBox {
       _resolve();
       final Size childSize = child.size;
       final FittedSizes sizes = applyBoxFit(_fit, childSize, size);
-      final double scaleX = sizes.destination.width / sizes.source.width;
-      final double scaleY = sizes.destination.height / sizes.source.height;
+      final double scaleX = _divHandleZero(sizes.destination.width, sizes.source.width);
+      final double scaleY = _divHandleZero(sizes.destination.height, sizes.source.height);
```

Comment:
Uber nit: 
```suggestion
      final double scaleX = _divisionHandleZero(sizes.destination.width, sizes.source.width);
```
to avoid abbreviations

Reply:
Thanks @dnfield and @Piinks for your comments! I've now dropped the `_devHandleZero`. Instead, I fixed the test and make `RenderFittedBox` to no longer paint with empty sizes. Add @jason-simmons for review as he wrote the test originally to fix a similar issue with NaN and empty size.